File: SamCross/ExampleProjects/Math_Graphic/src/main/java/core/Comon/Bitmap.java
package  core.Comon;

/**
 * Created by Michal on 2018-04-07.
 */
public class Bitmap {

    public int[][] yourArray; // tablica kontener danych na obiekty graficzne.
    public Bitmap(int size)
    {
        yourArray = new int[size][size];
    }
    public Bitmap(int array[][])
    {
        yourArray = array;
    }

    public int GetSize(){
        return yourArray.length;
    }

    public int get(int i, int j){
        return yourArray[i][j];
    }
    public int[][] getArray(){
        return yourArray;
    }

}

File: stryker-net/ExampleProjects/Math_Graphic/src/Math_Graphic/core/common/Bitmap.cs
namespace Math_Graphic.core.common;

public class Bitmap
{
    private readonly int[,] _yourArray;
    public Bitmap(int size)
    {
        _yourArray = new int[size, size];
    }
    public Bitmap(int [,] array)
    {
        _yourArray = array;
    }

    public int GetSize(){
        return _yourArray.Length;
    }

    public int Get(int i, int j){
        return _yourArray[i, j];
    }
    public int[,] GetArray(){
        return _yourArray;
    }
}